app/controllers/dogs_controller.rb

```diff
@@ -117,130 +115,112 @@ def destroy
-                                    :coordinator_id,
-                                    :sponsored_by,
-                                    :shelter_id,
-                                    :medical_summary,
-                                    attachments_attributes:
-                                    [
-                                      :attachment,
-                                      :description,
-                                      :updated_by_user_id,
-                                      :_destroy,
-                                      :id
-                                    ],
-                                    photos_attributes:
-                                    [
-                                      :photo,
-                                      :position,
-                                      :is_private,
-                                      :_destroy,
-                                      :id
-                                    ]
-
-                                  )
-    end
+                                    :updated_by_user_id,
+                                    :_destroy,
+                                    :id
+                                  ],
+                                  photos_attributes:
+                                  [
+                                    :photo,
+                                    :position,
+                                    :is_private,
+                                    :_destroy,
+                                    :id
+                                  ]
+
+                                )
+  end
 
-    def load_dog
-      @dog = Dog.find(params[:id])
-    end
+  def load_dog
+    @dog = Dog.find(params[:id])
+  end
 
-    def load_instance_variables
-      5.times { @dog.photos.build }
-      5.times { @dog.attachments.build }
-      @foster_users = User.where(is_foster: true).order("name")
-      @coordinator_users = User.where(edit_all_adopters: true).order("name")
-      @shelters = Shelter.order("name")
-      @breeds = Breed.all
-    end
+  def load_instance_variables
+    5.times { @dog.photos.build }
+    5.times { @dog.attachments.build }
+    @foster_users = User.where(is_foster: true).order("name")
+    @coordinator_users = User.where(edit_all_adopters: true).order("name")
+    @shelters = Shelter.order("name")
+    @breeds = Breed.all
+  end
 
-    def edit_dogs_user
-      redirect_to(root_path) unless current_user.edit_dogs?
-    end
+  def edit_dogs_user
+    redirect_to(root_path) unless current_user.edit_dogs?
+  end
 
-    def add_dogs_user
-      redirect_to(root_path) unless current_user.add_dogs?
-    end
+  def add_dogs_user
+    redirect_to(root_path) unless current_user.add_dogs?
+  end
 
-    def admin_user
-      redirect_to(root_path) unless current_user.admin?
-    end
+  def admin_user
+    redirect_to(root_path) unless current_user.admin?
+  end
 
-    def is_fostering_dog?(*args)
```

Comment:
this method was doing some weird stuff, take a look and make sure I didn't trim too much but it seems like its only job should be to compare the `@dog.foster_id` to the `current_user.id`. Also I saw no calling of this method with args, making that branch pretty much worthless.